core/validation/utils.py

```diff
@@ -12,3 +17,88 @@ def remove_duplicate_indexes(df: pd.DataFrame):
+    the start year of the UK financial year as each year represents an additional 5 rows in the
+    original spreadsheet from the row where the entry for a given footfall outcome's data begins.
+
+    :param failed_row: A pandas Series representing a row where an error has occurred.
+    :param index: The index indicating the position of the error in the data.
+    :return: A string containing the constructed cell index for outcomes.
+    """
+    start_date = failed_row["Start_Date"]
+
+    # footfall outcomes starts from row 60
+    if index >= 60:
+        financial_year = get_uk_financial_year_start(start_date)
+        # row for 'Amount' column is end number of start year of financial year * 5 + 'Footfall Indicator' index
+        row_index_gap = int(financial_year[-1]) * 5
+        index = str(int(index) + row_index_gap)
+        cell_index = footfall_outcome_amount_to_col(start_date).format(i=index)
+    else:
+        index = str(index)
+        cell_index = non_footfall_outcome_amount_to_col(start_date).format(i=index)
+
+    return cell_index
+
+
+def get_uk_financial_year_start(start_date: str) -> str:
+    """
+    Gets the start year of the UK financial year based on the provided start date.
+
+    :param start_date: A string representing the start date in the format '%Y-%m-%d %H:%M:%S'.
+    :return: A string representing the start year of the UK financial year.
+    """
+    start_date = datetime.strptime(start_date, "%Y-%m-%d %H:%M:%S")
+
+    financial_year_start = datetime(start_date.year, 4, 1)
+    if start_date < financial_year_start:
+        financial_year_start = datetime(start_date.year - 1, 4, 1)
+
+    return f"{str(financial_year_start.year)}"
+
+
+def get_month(start_date: str) -> str:
+    """
+    Gets the month name based on the provided start date.
+
+    :param start_date: A string representing the start date in the format '%Y-%m-%d %H:%M:%S'.
+    :return: A string representing the month name.
+    """
+    start_date = datetime.strptime(start_date, "%Y-%m-%d %H:%M:%S")
+
+    return calendar.month_name[start_date.month]
+
+
+def non_footfall_outcome_amount_to_col(start_date: str) -> str:
+    """
+    Maps the UK financial year start to the original column letter.
+
+    :param start_date: A string representing the start date in the format '%Y-%m-%d %H:%M:%S'.
+    :return: A string representing the original column letter corresponding to the financial year start.
+    """
+    to_map = get_uk_financial_year_start(start_date)
+
+    return FINANCIAL_YEAR_OR_MONTH_TO_ORIGINAL_COLUMN_LETTER_FOR_OUTCOMES[to_map]
+
+
+def footfall_outcome_amount_to_col(start_date: str) -> str:
+    """
+    Maps the month to the original column letter.
+
+    :param start_date: A string representing the start date in the format '%Y-%m-%d %H:%M:%S'.
+    :return: A string representing the original column letter corresponding to the month.
+    """
+    to_map = get_month(start_date)
```

Comment:
do we need a function for this? I think we can just do `calendar.month_name[date.month]`

Reply:
If we split into two dicts, with ints as keys then we can just do 
`MONTH_TO_FOOTFALL_OUTCOME_COLUMN[date.month]`